packages/fiori/src/themes/UploadCollectionItem.css

```diff
@@ -179,3 +179,7 @@
 [ui5-button]:last-of-type {
 	margin-inline-end: var(--ui5_upload_collection_last_button_inline_end);
 }
+
+:host-context([show-error-message]) .ui5-li-root.ui5-uci-root {
+	display: none;
+}
```

Comment:
Hides the content in the shadow root, but the ui5-upload-collection-item remains with `display: block` and it has height of "1px". If you show the error illustration on UC with mulptile items, you will see grey block.

When `showErrorMessage` is set, try not to render the default slot of the list at all